src/auth-service/routes/v2/users.js

```diff
@@ -769,4 +769,25 @@ router.get(
   createUserController.list
 );
 
+router.delete(
+  "/deleteMobileUserData",
+  oneOf([
+    body("email")
+      .optional()
```

Comment:
the input validation for `v1` is different from the one of `v2`, intentional? @BenjaminSsempala

Reply:
Thanks for catching this @Baalmart . I've corrected this